x-pack/plugins/apm/public/components/shared/service_icons/index.tsx

```diff
@@ -30,6 +31,18 @@ const cloudIcons: Record<string, string> = {
   azure: 'logoAzure',
 };
 
+const serverlessTitles: Record<string, string> = {
```

Comment:
Like here you could do:
```suggestion
const serverlessTitles: Record<ServerlessProvider | 'default', string> = {
```